src/api/listing/service.proto

```diff
@@ -201,13 +213,13 @@ message ListInventorySinceRequest {
   // Only return variants with inventory changes that have happened
   // outside of sales on this channel.
   bool external_changes_only = 2;
-  string cursor              = 3;  // TODO: how to get these in the query params
+  string cursor = 3; // TODO: how to get these in the query params
```

Comment:
doesn't seem like these are TODOs anymore?

Reply:
yea i guess not, just removed them